executor/travis/setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Sets up the travis worker to be able to run executor tests.
+
+export PROJECT_DIR=`pwd`
+cd ${PROJECT_DIR}
+
+pip install virtualenv
+virtualenv -p python3 py35
```

Comment:
Why do we need to use virtualenv for this?

Reply:
`python setup.py install` will attempt to download and install modules, we dont have write access to the standard python installation.

Reply:
👍. Nitpick: I think I would favor `venv` for the virtual env directory